tests/api_tests/collections/test_views.py

```diff
@@ -0,0 +1,622 @@
+        assert_not_in(self.private_collection_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.dashboard._id, ids)
+
+    def test_get_some_collections_with_substring_logged_in(self):
+        url = "/{}collections/?filter[title]=Two".format(API_BASE)
+
+        res = self.app.get(url, auth=self.basic_auth_one)
+        node_json = res.json['data']
+
+        ids = [node['id'] for node in node_json]
+        assert_not_in(self.collection_one._id, ids)
+        assert_in(self.collection_two._id, ids)
+        assert_not_in(self.collection_three._id, ids)
+        assert_not_in(self.private_collection_user_one._id, ids)
+        assert_not_in(self.private_collection_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.dashboard._id, ids)
+
+    def test_get_only_public_or_my_collections_with_filter_logged_in(self):
+        url = "/{}collections/?filter[title]=Project".format(API_BASE)
+
+        res = self.app.get(url, auth=self.basic_auth_one)
+        node_json = res.json['data']
+
+        ids = [node['id'] for node in node_json]
+        assert_in(self.collection_one._id, ids)
+        assert_in(self.collection_two._id, ids)
+        assert_not_in(self.collection_three._id, ids)
+        assert_in(self.private_collection_user_one._id, ids)
+        assert_not_in(self.private_collection_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.dashboard._id, ids)
+
+    def test_incorrect_filtering_field_logged_in(self):
+        # TODO Change to check for error when the functionality changes. Currently acts as though it doesn't exist
+        url = "/{}collections/?filter[notafield]=bogus".format(API_BASE)
+
+        res = self.app.get(url, auth=self.basic_auth_one)
+        node_json = res.json['data']
+
+        ids = [node['id'] for node in node_json]
+        assert_in(self.collection_one._id, ids)
+        assert_in(self.collection_two._id, ids)
+        assert_in(self.collection_three._id, ids)
+        assert_in(self.private_collection_user_one._id, ids)
+        assert_not_in(self.private_collection_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.dashboard._id, ids)
+
+
+class TestCollectionCreate(ApiTestCase):
+    def setUp(self):
+        ApiTestCase.setUp(self)
+        self.user = UserFactory.build()
+        self.user.set_password('justapoorboy')
+        self.user.save()
+        self.basic_auth = (self.user.username, 'justapoorboy')
+
+        self.url = "/{}collections/".format(API_BASE)
+
+        self.title = 'Cool Project'
+        self.category = 'data'
+
+        self.user_two = UserFactory.build()
+        self.user_two.set_password('justapoorboy')
+        self.user_two.save()
+        self.basic_auth_two = (self.user_two.username, 'justapoorboy')
+
+        self.public_folder = {'title': self.title, 'public': True, 'is_folder': True}
```

Comment:
Again, currently no such thing as a public collection, so remove these from the tests, and don't override the is_folder property in these tests. (It *should* be protected in the model, but it's not, so we have to just make sure the view doesn't allow it)